lib/apress/selenium_eti/spec/company_site/mini_eti/minieti_spec.rb

```diff
@@ -0,0 +1,200 @@
+        @product = @cs_eti_table_products.product(name: @name)
+      end
+
+      it 'введенное имя отображается' do
+        expect(@cs_eti_table_products.name(@product)).to eq(@name)
+      end
+
+      context 'когда добавляем картинку', skip: !RUN_CONFIG.fetch('local_running', false).to_b do
+        before(:all) do
+          @battery = @cs_eti_table_products.battery(@product)
+          @image = @cs_eti_table_products.image(@product)
+          @cs_eti_table_products.upload_image(@product, type: :local, path: IMAGE_PATH)
+        end
+
+        it 'картинка появляется' do
+          expect(@cs_eti_table_products.image(@product)[:url]).to be_truthy
+        end
+
+        it 'счетчик изображений увеличится на 1' do
+          expect(@cs_eti_table_products.image(@product)[:counts]).to eq(@image[:counts] + 1)
+        end
+
+        it 'увеличивается заряд батарейки' do
+          expect(@cs_eti_table_products.battery(@product)[:level])
+            .to eq(@battery[:level] + CONFIG['battery_percents']['image'])
+        end
+      end
+    end
+
+    context 'когда заполняем цену' do
+      before(:all) do
+        @name = Faker::Number.number(5)
+        @cs_eti_table_products.add_product(name: @name)
+        @product = @cs_eti_table_products.product(name: @name)
+        @battery = @cs_eti_table_products.battery(@product)
+        @price = {type: :exact, price: Faker::Number.number(3)}
+        @cs_eti_table_products.set_price(@product, @price)
+      end
+
+      it('введенная цена отображается') { expect(@cs_eti_table_products.price(@product)).to include(@price[:price]) }
+
+      it 'увеличивается заряд батарейки' do
+        expect(@cs_eti_table_products.battery(@product)[:level])
+          .to eq(@battery[:level] + CONFIG['battery_percents']['price'])
+      end
+    end
+
+    context 'когда заполняем цену от и до' do
+      before(:all) do
+        @name = Faker::Number.number(5)
+        @cs_eti_table_products.add_product(name: @name)
+        @product = @cs_eti_table_products.product(name: @name)
+        @battery = @cs_eti_table_products.battery(@product)
+        @price_range = {type: :range, price: Faker::Number.number(2), price_max: Faker::Number.number(3)}
+
+        @cs_eti_table_products.set_price(@product, @price_range)
+      end
+
+      it 'введенная цена отображается' do
+        expect(@cs_eti_table_products.price(@product)).to include @price_range[:price], @price_range[:price_max]
+      end
+    end
+
+    context 'когда заполняем цену со скидкой' do
+      before(:all) do
+        @name = Faker::Number.number(5)
+        @cs_eti_table_products.add_product(name: @name)
+        @product = @cs_eti_table_products.product(name: @name)
+        @battery = @cs_eti_table_products.battery(@product)
+        @discount_price = {
+          type: :discount,
+          price: Faker::Number.number(3),
+          new_price: Faker::Number.number(2),
+          expires_at: Time.now.strftime('%d.%m.%Y')
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.